database/01_table_creation.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE transaction_timestamps (
```

Comment:
Just occurred to me. Should we name it `transaction_timestamp` instead?

Reply:
Probably not strictly required anymore. I used some features only in the new version (psycopg) but the current code would also work with the old version (psycopg2).

I would generally say that we should use current stable version. psycopg (which is the new version, sometimes called psycopg 3) has been [out since 2021](https://pypi.org/project/psycopg/#history). The old version pycopg2 has [only received bug fixes since then](https://pypi.org/project/psycopg2/#history).

Reply:
The convention in the backend database seems to be to use the plural if individual entries are described in singular.

For the other tables, we also have a plural due to the one-to-many relation. Some of the code thus reads a bit strange.

I do not have a strong opinion here, though.

Reply:
`transaction_timestamp` is arguably more accurate as there is always exactly one. It is true that in the backend i see, e.g., `order_quotes` but i also see `order_execution`. So i am not sure if the backend db is consistent anyways

Reply:
I changed the name.